### PR TITLE
[Feature] Update to readme docs #3

### PR DESCRIPTION
**Pull request by Samsonroyal**
I am responding to issue #3 
[Improve README doc](https://github.com/oscakampala/oscakampala.github.io/issues/3)
as clearly stipulated in the above link. 

@KakoozaJerry please review the changes and see whether it's fit to be merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+
+![](.gitbook/assets/osca-logo_colored.png)
+
+You are welcome to our documentation guide; if you are looking to join **OSCA-Kampala Chapter** and want to learn more about **Free and Open Source Software** and also want to know how you can make a contribution to any of our projects. Read this documentaion as a guide; Although it is a work in progress, please feel free to contribute and give us feedback to improve it.
+
+To suggest changes, click the GitHub icon on any docs page and make a pull request. If you're not sure how to do that, read our [documentation contribution guidelines](https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/blob/main/CONTRIBUTING.md).
+
+If you need more help, please contact us at [community@oscakampala.org](mailto:community@oscakampala.org) or join the `#documentation` channel on our [Discord](https://discord.gg/pRJgjH9SwR) server.
```
Comment (KakoozaJerry) [inline README.md:11]: Our email is oscakampala@gmail.com, we currently do not have a domain of oscakampala.org

File: README.md [hunk 2]
```diff
@@ -1 +1,57 @@
...
+To get started in the open source there's a multitude of tools you can use but first of all you will need to know about OSCA Kampala. Read our **Introduction** section:
+
+[![Gitter](https://badges.gitter.im/OSCA-Kampala-Chapter/TTS.svg)](https://gitter.im/OSCA-Kampala-Chapter/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![License](https://img.shields.io/badge/License-MIT%202.0-brightgreen.svg)](https://opensource.org/licenses/MIT-2.0)
+
+📰 [**Subscribe to OSCA-KLA Newsletter**](https://www.getrevue.co/profile/oscakampala)
+
+🎧 [Anchor Podcasts](https://anchor.fm/osca-kampala) and [Google Podcasts](https://podcasts.google.com/feed/aHR0cHM6Ly9hbmNob3IuZm0vcy82ODg4NjU1NC9wb2RjYXN0L3Jzcw==)
+
+📄 [Text-to-Speech paper collection](https://github.com/erogol/TTS-papers)
```
Comment (KakoozaJerry) [inline README.md:24]: I think this is not supposed to be part of our readMe

**Review by KakoozaJerry — CHANGES_REQUESTED** [review]
I think this is great but some few comments I have made on specific section need to be addressed before I can merge this PR.

Thank you

File: README.md (hunk 1 above)
Comment (Samsonroyal) [reply README.md:11]: I think we need to buy that domain; it looks good on us; 
so what should I add to the domain name part in the meantime?

Let me make changes to the email address

File: README.md (hunk 2 above)
Comment (Samsonroyal) [reply README.md:24]: Noted; let me proceed to remove the two.

File: README.md (hunk 1 above)
Comment (KakoozaJerry) [reply README.md:11]: Just leave the domain name part blank I guess. for now, we can edit that later

File: README.md (hunk 2 above)
Comment (Samsonroyal) [reply README.md:24]: @KakoozaJerry should I do away with the entire How to get started section?

Comment (KakoozaJerry) [reply README.md:24]: @Samsonroyal Just the podcast link and text to speech link

File: README.md [hunk 3]
```diff
@@ -1 +1,57 @@
-This is a starter template for [Learn Next.js](https://nextjs.org/learn).
\ No newline at end of file
+# 📜 The Introduction
+
+**Open Source Community Kampala** is an organisation for open-source lovers, enthusiasts, advocates, and experts within and across Uganda but centered in Kampala with the sole aim of increasing the rate of credible contributions by Ugandan software developers, designers, writers, and everyone involved in the sphere of technology to open source projects both locally and globally, changing the perception the world has of our developers; we are not just users of software tools, we are also **Software creators**.
+
+![](.gitbook/assets/osca-logo_colored.png)
```
Comment (KakoozaJerry) [inline README.md:5]: Did we add the osca logo here?

Comment (Samsonroyal) [reply README.md]: I actually foresaw that and removed it; I have made a commit; kindly check and reconcile the facts.

**Review by KakoozaJerry — COMMENTED** [review]
So I will merge this but I still think this part should be more specific to how one can contribute to this repository with like steps on how to run this repository an their local machine and how to guides

**Pull request merged**